### DISCUSSION TITLE
[Feature] extend documentation on https://immich.app/docs/administration/jobs

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6218] [Feature Request]
### The feature

https://immich.app/docs/administration/jobs contains only the screenshot of what I can see in the GUI.
I would like to be documented, what are the different jobs doing. This would help to correct use different jobs.

EXTRACT METADATA
Extract metadata information from each asset, such as GPS and resolution
- does it check default and external libraries?
- does it check only new assets, or does it check also changed assets?
- what happens, when sidecare metadata exists?
- ...?

LIBRARY
Perform library tasks
- what kind of tasks are performed?

SIDECAR METADATA
Discover or synchronize sidecar metadata from the filesystem
- how this is related to "EXTRACT METADATA"?
- does it check default and external libraries?
- does it check only new assets, or does it check also changed assets?

and so on.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by aviv926** [comment]
Agreed, I think this part needs further expansion.
I tried to write down my information about facial recognition [here](https://github.com/immich-app/immich/pull/5958#discussion_r1438269821) (although I'm not sure of the level of accuracy regarding what I wrote) but it seems that a separate PR should be opened for this topic.

**Reply by aisbergde** [reply]
This kind of information would be useful in the docs. It't a good starting point to bring some light into the black box. I hope, that the docs will grow.
Thank you for this link! It's not about the jobs, but interested used can get a better understanding, what happens with face detection and recognition.
